R/write_codemeta.R

```diff
@@ -74,4 +79,16 @@ write_codemeta <- function(
   # Create or update codemeta and save to disk
   create_codemeta(pkg = pkg, root = root, use_filesize = use_filesize) %>%
     jsonlite::write_json(path, pretty = TRUE, auto_unbox = TRUE, ...)
+
+  # Create minimeta and save to disk
+  if (write_minimeta) {
+    if (!requireNamespace("jsonld", quietly = TRUE)) {
+      stop("Package jsonld required. Please install before re-trying.")
+    }
+    schemaorg <- system.file("schema", "schemaorg.json",
```

Comment:
Um, I'm not seeing an `inst/schema/schemaorg.json` file for the frame... Where's the frame live?